tools/device_file_generator/stm_writer.py

```diff
@@ -194,6 +194,7 @@ def addGpioToNode(self, node):
 					gpio_child = driver.addChild('gpio')
 					gpio_child.setAttributes(attr)
 					gpio_child.setAttributes(gpio)
+					
```

Comment:
Good question, the links to the [coding conventions in our CONTRIBUTING.md file](https://github.com/roboterclubaachen/xpcc/blob/develop/CONTRIBUTING.md) are all broken.

The old `coding_conventions.rst` file that can still be found in the git history had the following to say about white spaces:

> Remove trailing whitespace. Most editors can be configured to automatically
  strip trailing whitespace on file save, so use that option.